kura/distrib/src/main/resources/aarch64-nn/kura_install.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/sh
+#
+# Copyright (c) 2011, 2014, 2017 Eurotech and/or its affiliates
+#
+#  All rights reserved. This program and the accompanying materials
+#  are made available under the terms of the Eclipse Public License v1.0
+#  which accompanies this distribution, and is available at
+#  http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#   Eurotech
+#
+
+INSTALL_DIR=/opt/eclipse
+
+#create known kura install location
+ln -sf ${INSTALL_DIR}/kura_* ${INSTALL_DIR}/kura
+
+#set up Kura init
+cp ${INSTALL_DIR}/kura/install/kura.init.raspbian /etc/init.d/kura
+chmod +x /etc/init.d/kura
+chmod +x ${INSTALL_DIR}/kura/bin/*.sh
+
+# setup snapshot_0 recovery folder
+if [ ! -d ${INSTALL_DIR}/kura/.data ]; then
+    mkdir ${INSTALL_DIR}/kura/.data
+fi
+
+# setup /etc/sysconfig folder for iptables configuration file
+if [ ! -d /etc/sysconfig ]; then
+    mkdir /etc/sysconfig
+fi
+
+#set up default networking file
+cp ${INSTALL_DIR}/kura/install/network.interfaces.raspbian /etc/network/interfaces
```

Comment:
Since this is an NN profile, I believe this set of changes that affect networking are not needed. Please refer, for example to an installer taken by another NN profile.